app/models/foreman_tasks/triggering.rb

```diff
@@ -93,10 +98,16 @@ def parse_start_before!
 
     private
 
-    def correct_cronline
-      ForemanTasks::RecurringLogic.new_from_cronline(cronline).next_occurrence_time
-    rescue ArgumentError => _
-      self.errors.add(:cronline, _("%s is not valid format of cron line")) % (cronline)
+    def can_start_recurring
+      recurring_logic.next_occurrence_time
```

Comment:
this line doesn't seem to be necessary, `valid?` nor `valid_cronline?` seems to rely on @parser being set, `valid_cronline?` calls it on its own